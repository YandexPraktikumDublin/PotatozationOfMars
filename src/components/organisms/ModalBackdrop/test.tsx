import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { ModalBackdrop } from '.'

describe('<ModalBackdrop />', () => {
  const children = 'Test children'

  it('should renders correct <ModalBackdrop />', () => {
    const wrapper = shallow(<ModalBackdrop>{children}</ModalBackdrop>)

    expect(wrapper.contains(children)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
