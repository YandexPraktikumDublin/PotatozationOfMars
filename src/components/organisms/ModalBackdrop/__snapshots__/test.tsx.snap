// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModalBackdrop /> should renders correct <ModalBackdrop /> 1`] = `
<Portal
  containerInfo={
    <div
      class="fixed z-40 inset-0 overflow-y-auto"
    />
  }
>
  <div
    className="relative z-50 w-full h-full flex"
  >
    Test children
  </div>
  <div
    className="bg-black fixed inset-0 z-auto opacity-50"
  />
</Portal>
`;
